task/task-pangolin-ropsten/src/service/ropsten/ropsten_logs_handler.rs

```diff
@@ -79,7 +79,12 @@ impl LogsHandler for RopstenLogsHandler {
         let check_position = if txs.is_empty() { to } else { from - 1 };
         self.check_redeem(check_position).await?;
         if self.waited_redeem.len() >= MAX_WAITED_REDEEM_COUNT {
-            // todo: there need stop scan running
+            info!(
+                target: PangolinRopstenTask::NAME,
+                "achieve redeem waited limit, pause scan, from: {:?}",
+                &from
+            );
+            self.tracker.flush_current(from as usize)?;
```

Comment:
Just a question, how does this pause the scan?

`flush_current` just update some key in the db

Reply:
each time to scan, the tracker will read current from db during fast mode. So, if the current paused at `from` position, the tracker will always scan from this position.